definitions/npm/prismic.io_v3.x.x/flow_>=v0.28.x/prismic.io_v3.x.x.js

```diff
@@ -1,5 +1,5 @@
 declare module 'prismic.io' {
-  declare type linkResolver = (...args: any) => any;
+  declare type LinkResolver = (...args: any) => any;
```

Comment:
Probably ok here.

More generally: flow-typed libdefs are meant to be checked in, and upgrades are meant to be an actionable event by the developer (something like `flow-typed update`) and is never implicit. We should probably set expectations around updating libdefs that doing so may require some changes (i.e. a libdef can be improved to find new kinds of things).